src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -87,7 +87,7 @@ public void TestDriveProperties_AppContainer()
 
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Accessing drive format is not permitted inside an AppContainer.")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Accessing drive format is not permitted inside an AppContainer.")]
```

Comment:
Was this just incorrectly skipped in uapaot since we are not running on appcontainer in that case?